Source/JavaScriptCore/wasm/js/WasmToJS.cpp

```diff
@@ -61,7 +61,8 @@ static Expected<MacroAssemblerCodeRef<WasmEntryPtrTag>, BindingFailure> handleBa
     jit.loadPtr(CCallHelpers::Address(GPRInfo::argumentGPR2, Instance::offsetOfOwner()), GPRInfo::argumentGPR0);
     jit.loadPtr(CCallHelpers::Address(GPRInfo::argumentGPR0, JSWebAssemblyInstance::offsetOfModule()), GPRInfo::argumentGPR1);
     jit.prepareCallOperation(vm);
-    jit.emitPutToCallFrameHeader(GPRInfo::argumentGPR1, CallFrameSlot::callee);
+    CCallHelpers::Address calleeSlot { GPRInfo::callFrameRegister, CallFrameSlot::callee  * sizeof(Register) };
```

Comment:
Are all uses of jit.emitPutToCallFrameHeader wrong on ARMv7 then?

Reply:
Should we change that function instead?

Reply:
I don't think this applies to all instances of that function call, because it's used on different slots in the call frame. It's specifically the callee slot that's the issue here, and it's used on `CallFrameSlot::codeBlock` instead for example.

There might be ~2 other calls that would be good to check on though.

Reply:
Maybe we can make a helper, putCellToFrame?

Reply:
I added a helper in the latest version, and refactored one other case in `WasmToJS.cpp` to use the helper (there is another bit of code modifying the cell tag too, but it's touching the callee frame so the helper doesn't apply).